internal/operands/common-templates/reconcile_test.go

```diff
@@ -0,0 +1,80 @@
+package common_templates
+
+import (
+	"context"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"testing"
+
+	ssp "kubevirt.io/ssp-operator/api/v1alpha1"
+	"kubevirt.io/ssp-operator/internal/common"
+)
+
+var log = logf.Log.WithName("common-templates-operand")
+
+func TestTemplates(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Template Validator Suite")
+}
+
+var _ = Describe("Common-Templates operand", func() {
+	const (
+		namespace = "kubevirt"
+		name      = "test-ssp"
+	)
+
+	var request common.Request
+
+	BeforeEach(func() {
+		s := scheme.Scheme
+		Expect(ssp.AddToScheme(s)).ToNot(HaveOccurred())
+
```

Comment:
The templates must also be added to schema here:
```
Expect(AddWatchTypesToScheme(s)).ToNot(HaveOccurred())
```

Reply:
Added